### DISCUSSION TITLE
How should data be displayed when it is available both for smaller regions and a larger area?

### DISCUSSION THREAD

**Discussion by q--** [electricitymaps/electricitymaps-contrib#2759] [Q&A] [answered]
E.g., data is available for both the [regions of France](https://github.com/tmrowco/electricitymap-contrib/search?q=regions+france&type=issues) and France as a whole. Same goes for Sweden. As for India, there's data for some of the Indian states and the country as a whole.

It would be nice to be able to display data for both on ElectricityMap, but as these areas -obviously- overlap, ElectricityMap currently has to choose. In most cases this isn't a problem, but in the case of India both the national and state-based data has flaws or limitations (e.g. not knowing the exact fuel of thermal power plants, or missing imports and exports). France has no real-time power flows between regions (it shows imports, but not from which region the electricity is imported).

How should data be displayed in these cases?

**Comment by Kongkille** [comment]
It's a great question. Having multiple views would indeed help us with coverage, however, I have a few concerns.

I think UX-wise this is going to be extremely difficult to display. It's going to be hard not to favor one type of view or make it quite unintuitive for a user to understand what's going on. Did you have any thoughts about this? Technically we should also consider the flow-tracing of the model because in this case, for example, which subzones would be responsible for the import/export.

**Comment by q--** [comment, marked as answer]
This is now mostly implemented by having a country/region toggle.

Also see #3309
